eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -65,6 +65,11 @@ class PackageProps
 }
 
 $ProgressPreference = "SilentlyContinue"
+
+if ((Get-PSRepository -WarningAction:SilentlyContinue) -eq $null)
+{
+    Register-PSRepository -Default
```

Comment:
What happens if we always blindly call Register-PSRegistory? Will it be a no-op if it is already registered? If so lets just always call it.

Reply:
Looks like you might be able to do something like `Register-PSRepository -Default -ErrorAction:SilentlyContinue`